cmd/minikube/cmd/config/prompt.go

```diff
@@ -55,9 +56,14 @@ func AskForYesNoConfirmation(s string, posResponses, negResponses []string) bool
 }
 
 // AskForStaticValue asks for a single value to enter
-func AskForStaticValue(s string) string {
+func AskForStaticValue(s string, isOptional ...bool) string {
 	reader := bufio.NewReader(os.Stdin)
 
+	optional := false
+	if len(isOptional) > 0 {
```

Comment:
This is setting optional to true if one or more bools are passed in since `isOptional` is an array of bools. For example the following are all optional prompts
``` go
AskForStaticValue("example 1", true)
AskForStaticValue("example 2", false)
AskForStaticValue("example 3", true, false)
```
Is this the expected behavior?